packages/grafana-ui/src/components/Logs/LogRowContextProvider.tsx

```diff
@@ -101,11 +100,7 @@ export const getRowContexts = async (
           const lineField: Field<string> = dataFrame.fields.filter(field => field.name === 'line')[0];
           const line = lineField.values.get(fieldIndex); // assuming that both fields have same length
 
-          if (data.length === 0) {
-            data[0] = [line];
-          } else {
-            data[0].push(line);
-          }
+          data.push(line);
```

Comment:
Nice! Do we know if there was any specific reason to create that extra layer? As it doesn't seem like a mistake, as it is clearly visible in tests. Just curious. 🤔

Reply:
Not exactly sure, but probably it was introduced with changes in type definitions about a year ago.